Library/Homebrew/system_config.rb

```diff
@@ -210,7 +210,7 @@ def dump_verbose_config(f = $stdout)
       f.puts "Homebrew Ruby: #{describe_homebrew_ruby}"
       f.puts "GCC-4.0: build #{gcc_4_0}" unless gcc_4_0.null?
       f.puts "GCC-4.2: build #{gcc_4_2}" unless gcc_4_2.null?
-      f.puts "Clang: #{clang.null? ? "N/A" : "#{clang} build #{clang_build}"}"
+      f.puts "Clang: #{clang.null? ? "N/A" : "#{clang} build #{clang_build.null? ? "???" : clang_build}"}"
```

Comment:
The intent is to make it more noticeable when the above regex doesn't match. I think it does a good job of that.
The output in that case looks like so:

```
Clang: 5.0 build ???
```

Reply:
I'd suggest `Clang: 5.0 build N/A`.

Reply:
"Not applicable" is the correct answer when a tool is not installed, which is its usage in the rest of this file, but the build number is indeed applicable when clang is installed, as it is here.

Reply:
Fair enough. I don't feel strongly.

Reply:
<3

Reply:
I changed the "???" string to "(parse error)", which should be more clear.